hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/PopulateJMSMessageHeader.java

```diff
@@ -14,13 +14,17 @@
  * Add hexadecimal message ControlId with a prefix ID: to add provider specific 
  * value to stop any encoding in MQRFH2.
  * Set custom JMS message Id by adding MQMD.
+ * Set CCSID to 819
+ * Set DeliveryMode to 'MQPER_NOT_PERSISTENT
```

Comment:
I assume MQPER_NOT_PERSISTENT is code 1 but might want to make that clear in the comment. Also there's only a opening single quote and no closing quote.